\hypertarget{classneo4j_1_1work_1_1pipelining_1_1_pull_order_exception}{}\doxysection{neo4j.\+work.\+pipelining.\+Pull\+Order\+Exception Class Reference}
\label{classneo4j_1_1work_1_1pipelining_1_1_pull_order_exception}\index{neo4j.work.pipelining.PullOrderException@{neo4j.work.pipelining.PullOrderException}}


Inheritance diagram for neo4j.\+work.\+pipelining.\+Pull\+Order\+Exception\+:
% FIG 0


Collaboration diagram for neo4j.\+work.\+pipelining.\+Pull\+Order\+Exception\+:
% FIG 1


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raise when calling pull if a previous pull result has not been fully consumed\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
venv/\+Lib/site-\/packages/neo4j/work/pipelining.\+py\end{DoxyCompactItemize}
